src/Microsoft.Fast.Components.FluentUI/Components/Slider/FluentSliderLabel.razor.cs

```diff
@@ -26,4 +27,20 @@ public partial class FluentSliderLabel : FluentComponentBase
     /// </summary>
     [Parameter]
     public RenderFragment? ChildContent { get; set; }
+
+    protected string? FormatValueAsString(TValue? value)
+    {
+        // Avoiding a cast to IFormattable to avoid boxing.
+        return value switch
```

Comment:
This code is also used in FluentSlider. Could you add a common method in the `InputHelpers` class to centralize these conversions?

Reply:
Done. Using it in `FluentNumberField` now as well